openai-chatbot/src/app/layout.tsx

```
@@ -0,0 +1,42 @@
+import type { Metadata } from "next";
+import { Geist, Geist_Mono } from "next/font/google";
+import "./globals.css";
+
+import { Geist, Geist_Mono } from "next/font/google";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Geist'. Consider to delete it or rename it.

'Geist' is defined here:


(lint/suspicious/noRedeclare)